diff --git a/node_modules/svelty-picker/dist/components/SveltyPicker.svelte b/node_modules/svelty-picker/dist/components/SveltyPicker.svelte
index fc6354a..751d4ed 100644
--- a/node_modules/svelty-picker/dist/components/SveltyPicker.svelte
+++ b/node_modules/svelty-picker/dist/components/SveltyPicker.svelte
@@ -1,5 +1,5 @@
 <script context="module">
-  import settings from "../settings";
+  import settings from "../settings.js";
   export const config = settings;
 </script>
 
@@ -12,10 +12,10 @@
   import { fade } from "svelte/transition";
   import Calendar from "./Calendar.svelte";
   import Time from "./Time.svelte";
-  import { formatDate, parseDate } from "../utils/dateUtils";
+  import { formatDate, parseDate } from "../utils/dateUtils.js";
   import { usePosition } from "../utils/actions.js";
-  import { computeResolvedMode, initProps } from "../utils/state";
-  import { MODE_MONTH, STARTVIEW_TIME } from "../utils/constants";
+  import { computeResolvedMode, initProps } from "../utils/state.js";
+  import { MODE_MONTH, STARTVIEW_TIME } from "../utils/constants.js";
 
   // html
   export let inputId = '';
diff --git a/node_modules/svelty-picker/dist/utils/grid.js b/node_modules/svelty-picker/dist/utils/grid.js
index 19c3403..5318956 100644
--- a/node_modules/svelty-picker/dist/utils/grid.js
+++ b/node_modules/svelty-picker/dist/utils/grid.js
@@ -1,6 +1,6 @@
 
-import { MODE_YEAR, MODE_DECADE } from "./constants";
-import { getDaysInMonth } from "./dateUtils";
+import { MODE_YEAR, MODE_DECADE } from "./constants.js";
+import { getDaysInMonth } from "./dateUtils.js";
 
 /**
  * @typedef {object} Dataset
diff --git a/node_modules/svelty-picker/dist/utils/state.js b/node_modules/svelty-picker/dist/utils/state.js
index e264172..c6101a2 100644
--- a/node_modules/svelty-picker/dist/utils/state.js
+++ b/node_modules/svelty-picker/dist/utils/state.js
@@ -1,4 +1,4 @@
-import { formatDate, parseDate } from "./dateUtils";
+import { formatDate, parseDate } from "./dateUtils.js";
 
 /**
  * @typedef {object} ValueInit
