diff --git a/node_modules/tailwindcss/lib/public/colors.js b/node_modules/tailwindcss/lib/public/colors.js
index 6d85878..6e219c3 100644
--- a/node_modules/tailwindcss/lib/public/colors.js
+++ b/node_modules/tailwindcss/lib/public/colors.js
@@ -311,45 +311,5 @@ const _default = {
         800: "#9f1239",
         900: "#881337",
         950: "#4c0519"
-    },
-    get lightBlue () {
-        warn({
-            version: "v2.2",
-            from: "lightBlue",
-            to: "sky"
-        });
-        return this.sky;
-    },
-    get warmGray () {
-        warn({
-            version: "v3.0",
-            from: "warmGray",
-            to: "stone"
-        });
-        return this.stone;
-    },
-    get trueGray () {
-        warn({
-            version: "v3.0",
-            from: "trueGray",
-            to: "neutral"
-        });
-        return this.neutral;
-    },
-    get coolGray () {
-        warn({
-            version: "v3.0",
-            from: "coolGray",
-            to: "gray"
-        });
-        return this.gray;
-    },
-    get blueGray () {
-        warn({
-            version: "v3.0",
-            from: "blueGray",
-            to: "slate"
-        });
-        return this.slate;
     }
 };
